api/web3server.go

```diff
@@ -837,6 +796,58 @@ func (svr *Web3Server) getFilterLogs(in *gjson.Result) (interface{}, error) {
 	return svr.getLogsWithFilter(from, to, filterObj.Address, filterObj.Topics)
 }
 
-func (svr *Web3Server) unimplemented() (interface{}, error) {
+func (svr *web3Handler) subscribe(in *gjson.Result, writer apitypes.JSONWriter) (interface{}, error) {
+	if writer.IsWriteOnce() {
```

Comment:
why we need this check?

Reply:
eth_subscribe is prohibited in http server